lib/tree-view.coffee

```diff
@@ -401,26 +432,26 @@ class TreeView extends View
     selectedEntry = @selectedEntry()
     return unless selectedEntry?
 
-    if directory = $(selectedEntry).closest('.expanded.directory')[0]
+    if directory = selectedEntry.closest('.expanded.directory')
       directory.collapse(isRecursive)
       @selectEntry(directory)
 
   openSelectedEntry: (options={}, expandDirectory=false) ->
     selectedEntry = @selectedEntry()
-    if selectedEntry instanceof DirectoryView
+    if selectedEntry.classList.contains('directory')
```

Comment:
Fixed in a7e7be1. @Alhadis: thanks so much for finding these pesky regressions! ⚡️